src/scss/_sofie-page.scss

```diff
@@ -53,28 +23,67 @@
     width: 100%;
     font-size: 1rem;
   }
+  & mat-form-field {
+    position: relative;
+    width: 100%;
+    & .mat-form-field-wrapper {
+      border-radius: 4px;
+      border: 1px solid var(--gray-color);
+      padding-bottom: 0;
+      & .mat-form-field-infix {
+        border-top: 0;
+        padding: 0.5rem 1rem;
+      }
+    }
+    & .mat-form-field-underline {
+      display: none;
+    }
+  }
+}
+
+.c-settings-title {
+  margin: 30px;
+  font-size: 2rem;
+  font-weight: lighter;
 }
 
 .c-settings-page {
-  padding: 32px;
+  padding: 0;
```

Comment:
As mentioned in a previous comment, we should have reuseable components for the scafolding of our Settings pages instead of having global CSS classes. 
We should aim to limit our use of global CSS classes as much as possible.